corehq/toggles.py

```diff
@@ -891,6 +891,16 @@ def _enable_search_index(domain, enabled):
                 "This may still cause issues if the other questions have not been calculated yet",
 )
 
+VELLUM_ALLOW_BULK_FORM_ACTIONS = StaticToggle(
+    'allow_bulk_form_actions',
+    "Allow bulk form actions in Vellum / Formdesigner",
+    TAG_PRODUCT,
+    [NAMESPACE_DOMAIN],
+    description="This shows Bulk Form Actions (mark all questions required, "
+                "set default values to matching case properties) in "
+                "Vellum's main dropdown menu.",
```

Comment:
I think I would remove the name "Vellum" from this. Except among coders it's normally called "Form Builder", right?
```suggestion
    "Allow bulk form actions in the Form Builder",
    TAG_PRODUCT,
    [NAMESPACE_DOMAIN],
    description="This shows Bulk Form Actions (mark all questions required, "
                "set default values to matching case properties) in "
                "the Form Builder's main dropdown menu.",
```